templates/account.md

```diff
@@ -98,6 +98,58 @@ The account links tile is simply an array of docIds for [account-links](./doctyp
 
 
 
+### Claims Tile
+
+The claims tile contains a list of claims issued by third party services about the user. For example this could include social verifications for twitter, discord, and discourse. 
+
+##### Schema
+
+```json
+{
+    "$schema": "http://json-schema.org/draft-06/schema#",
+    "type": "array",
+    "items": {
+        "type": "string"
+    },
+    "definitions": {}
+}
+```
+
+##### Example
+
+```jsonc
+[<JWT>, <JWT>]
+```
+
+
+
+### Connections Tile
```

Comment:
@oed I think we want this to include pointers to a `following` tile (public) and a `contacts` tile (encrypted).

Reply:
This allows this connections to be the abstraction, with various implementations of it.

Reply:
I assumed this would be public only. Besides you probably don't need to timestamp your private contact list on the blockchain. Could be stored in a threadsdb or orbitdb instead.

Reply:
Although, it might be interesting with some form of semi-private contact lists.